internal/pkg/util/auth/token.go

```diff
@@ -22,7 +22,7 @@ const (
 	// WarningEmptyToken Warning return for empty token string
 	WarningEmptyToken = "Token file is empty"
 	// WarningTokenFileNotFound token file not found
-	WarningTokenFileNotFound = "Authentication token file not found"
+	WarningTokenFileNotFound = "Authentication token file not found. To generate a token, visit cloud.sylabs.io/auth"
```

Comment:
Do you want to put in a leading  `https://`  here?

Reply:
No, it gets filled in automatically by any modern browser afaik

Reply:
Default will be `http` ... The front end will redirect to `https` ... This is just to be explicit.

Reply:
Simplicity is good and but URLs in many terminals are clickable when they are prefixed properly. Let's prefix with https://.